manim/utils/config_utils.py

```diff
@@ -135,6 +135,17 @@ def _parse_file_writer_config(config_parser, args):
         [fw_config["save_last_frame"], fw_config["from_animation_number"]]
     )
 
+    # the the custom_folders flag overrides the default folder structure with
+    # the custom folders defined in default.cfg
+    if args.custom_folders:
+        from pathlib import Path
+
+        root_folder = str(config_parser["custom_folders"].get("custom_root"))
+        for opt in ["video_dir", "tex_dir", "text_dir", "output_file"]:
+            sub_folder = config_parser["custom_folders"].get(opt)
+            path = Path.home() / root_folder / sub_folder
+            fw_config[opt] = str(path)
+        fw_config["output_file"] = fw_config["output_file"] + "/SceneName"
```

Comment:
This part  "/SceneName" is only a placeholder for now, do you know how I can get the actual SceneName here?

Reply:
You don't need to. If an output_file name is not given, manim will use the name of the scene and put it in the videos folder. Please try/test your code after removing line 148 and make sure that it works.

Reply:
Removing this line gives me this error:Traceback (most recent call last):
```
  File "/home/jan-hendrik/projects/manim-community/manim/__main__.py", line 184, in main
    scene = SceneClass()
  File "/home/jan-hendrik/projects/manim-community/manim/scene/scene.py", line 75, in __init__
    self.file_writer.finish()
  File "/home/jan-hendrik/projects/manim-community/manim/scene/scene_file_writer.py", line 353, in finish
    self.combine_movie_files()
  File "/home/jan-hendrik/projects/manim-community/manim/scene/scene_file_writer.py", line 448, in combine_movie_files
    partial_movie_files = get_sorted_integer_files(
  File "/home/jan-hendrik/projects/manim-community/manim/utils/file_ops.py", line 60, in get_sorted_integer_files
    os.remove(full_path)
IsADirectoryError: [Errno 21] Is a directory: '/home/jan-hendrik/Videos/temp_files/scratch_pathlib_folder_Stucture'
```

Reply:
This issue is due to l.69, `file_ops.py`. basically, manim removed (this is not the case now with scene-caching) every file whose name did not match an integer index name (eg 0001.mp4, etc..). 
As in temp_file there are a lot of different stuff (here the video generated, under the form of SceneName/resolution/video.mp4) manim _thinks_ it should be removed but can't as it's not a file but a directory. 

If you merge this from master it will be fixed. (in theory)